pkg/controller/migration/convert/convert_test.go

```diff
@@ -6,46 +6,63 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	"github.com/tigera/operator/pkg/apis"
+	crdv1 "github.com/tigera/operator/pkg/apis/crd.projectcalico.org/v1"
 	operatorv1 "github.com/tigera/operator/pkg/apis/operator/v1"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
 var _ = Describe("Parser", func() {
 	var ctx = context.Background()
+	var pool *crdv1.IPPool
+	var scheme *runtime.Scheme
+	BeforeEach(func() {
+		scheme = kscheme.Scheme
+		err := apis.AddToScheme(scheme)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
You might be able to do this in a `BeforeSuite` in the suite initializer. But I know how test initialization with ginkgo can be a bit of a rabbit hole so happy for you to just leave as is.